Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -1796,7 +1799,6 @@ void WebPageProxy::close()
     internals().audibleActivityTimer.stop();
 
     stopAllURLSchemeTasks();
-    updatePlayingMediaDidChange(MediaProducer::IsNotPlaying);
```

Comment:
I removed this because it seemed like it was doing nothing--we are already calling it in `resetState` above.